test/unit/salt_runner_test.rb

```diff
@@ -0,0 +1,51 @@
+if RUBY_VERSION >= '2.3.0'
```

Comment:
Haven't we dropped Ruby < 2.3? Or is this a good excuse to do so? Smart Proxy itself is already Ruby 2.5+.